scripts/us_census/pep/population_estimate_by_race/preprocess.py

```diff
@@ -0,0 +1,936 @@
+    cols = df.columns.drop(extras)
+    df[cols] = df[cols].apply(pd.to_numeric, errors='coerce')
+    # All the age groups are being added up to get total value.
+    df['count']=df['Under 5 years']+df['5 to 9 years']+df['10 to 14 years']\
+        +df['15 to 19 years']+df['20 to 24 years']+df['25 to 29 years']+\
+        df['30 to 34 years']+df['35 to 39 years'] +df['40 to 44 years']	\
+        +df['45 to 49 years'] +df['50 to 54 years'] +df['55 to 59 years']\
+        +df['60 to 64 years'] +df['65 to 69 years'] +df['70 to 74 years']\
+        +df['75 to 79 years'] +df['80 to 84 years'] +df['85 years and over']
+    df = df.drop(['Under 5 years','5 to 9 years','10 to 14 years',\
+        '15 to 19 years','20 to 24 years','25 to 29 years',\
+        '30 to 34 years','35 to 39 years','40 to 44 years',\
+        '45 to 49 years','50 to 54 years','55 to 59 years',\
+        '60 to 64 years','65 to 69 years','70 to 74 years','75 to 79 years',\
+        '80 to 84 years','85 years and over'],axis=1)
+    df['locationyear'] = df['Year of Estimate'] + "-" + df['State Name']
+    df = df.drop(['Year of Estimate', 'State Name'], axis=1)
+    # it groups the df as per columns provided
+    # performs the provided functions on the data
+    # The rows and columns have been transposed as per requirements
+    df = df.groupby(['locationyear','Race']).sum().transpose().\
+        stack(0).reset_index()
+    df['Year'] = df['locationyear'].str.split('-', expand=True)[0]
+    df['Geographic Area'] = df['locationyear'].str.split('-', expand=True)[1]
+    df['Total'] = df["White"] + df["Black"] + df['Other races']
+    df = df.drop(['locationyear', 'level_0', 'Other races'], axis=1)
+    df.columns = df.columns.str.replace('White', 'White Alone')
+    df.columns = df.columns.str.replace('Black',
+                                        'Black or African American Alone')
+    return df
+
+
+def _clean_csv_file(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    This method cleans the dataframe loaded from a csv file format.
+    Also, Performs transformations on the data.
+
+    Arguments:
+        df (DataFrame) : DataFrame of csv dataset
+
+    Returns:
+        df (DataFrame) : Transformed DataFrame for csv dataset.
+    The values are comma seperated.
+    Created columns as per the name and dropped the
+    remaining columns.
+    """
+    for col in df.columns:
+        df[col] = df[col].astype(str)
+        df[col] = df[col].str.replace(",", "")
+    df['Total'] = pd.to_numeric(df['1'])
+    df['White Alone'] = pd.to_numeric(df['4'])
+    col = df.columns
+    if len(col) >= 15:
+        df['Black or African American Alone'] = pd.to_numeric(df['7'])
+        df = df.drop(["10", "11", "12"], axis=1)
+    else:
+        df['Non White'] = df['7']
+    df = df.drop(["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"], axis=1)
+    return df
+
+
+def _clean_county_20_csv_file(file_path: str) -> pd.DataFrame:
```

Comment:
Please add in doc string - which files this function is processing.

Reply:
Mentioned the files being cleaned.